controllers/front/CartController.php

```diff
@@ -614,10 +627,9 @@ protected function areProductsAvailable()
             $currentProduct->hydrate($product);
 
             if ($currentProduct->hasAttributes() && $product['id_product_attribute'] === '0') {
-                return $this->trans(
-                   'The item %product% in your cart is now a product with attributes. Please delete it and choose one of its combinations to proceed with your order.',
-                    ['%product%' => $product['name']],
-                    'Shop.Notifications.Error'
+                return $this->translateProductNameWithAttributes(
+                   'The product [1]%product%[/1] in your cart is now a product with attributes. Please delete it and choose one of its combinations to proceed with your order.',
```

Comment:
I wonder why it is written: "is NOW a product with attributes". 🤔 Wasn't that already the case?

Reply:
It's rare, but can happen, the customer add the product (standard type) in the cart, meanwhile the merchant modify the product in a product with combination, now that product isn't a standard product anymore 😊